xml_inscriptions/transcribed/CA.Malibu.JPGM.G.78.AA.58.xml

```diff
@@ -70,10 +70,28 @@
                   </summary>
                   <origin>
                      <!--Should contain information about origin of inscription including place and date.-->
-                     <date evidence="unspecified" notBefore="-0350" notAfter="-0350">c. 350 BCE</date>
+                     <date evidence="unspecified" notBefore="-0360" notAfter="-0360">c. 360 BCE</date>
```

Comment:
For approximate date ranges, typically you want to give some years before and some after in the `@notBefore` and `@notAfter` attributes.